content/spin/dynamic-configuration.md

```diff
@@ -0,0 +1,137 @@
+title = "Dynamic and Runtime Application Configuration"
+template = "spin_main"
+date = "2022-03-14T00:22:56Z"
```

Comment:
I don't see the `enable_shortcodes = true` here.
Perhaps I am confused because I thought `enable_shortcodes = true` was necessary for the `<!-- @nocpy -->` and `<!-- @selectiveCpy -->` to work. But this functionality seems to be working on this page during my local preview without the frontmatter part.

![Screenshot 2023-03-02 at 11 01 27 am](https://user-images.githubusercontent.com/9831342/222303923-90b1ca2b-33f7-45e5-a9b5-5cb4901dcb5c.png)

@karthik2804 what am I missing here? :)

Reply:
You don't need the enable shortcodes for the code block annotations. Currently, it is only required for `multi-block` content and `details` shortcodes.

Reply:
Ah, for multi-language codeblock; got it!
Thanks